diff --git a/libxerces-c/xercesc/util/KVStringPair.hpp b/libxerces-c/xercesc/util/KVStringPair.hpp
index aeb787a..87e2947 100644
--- a/libxerces-c/xercesc/util/KVStringPair.hpp
+++ b/libxerces-c/xercesc/util/KVStringPair.hpp
@@ -163,16 +163,6 @@ inline XMLCh* KVStringPair::getValue()
 // ---------------------------------------------------------------------------
 //  KVStringPair: Setters
 // ---------------------------------------------------------------------------
-inline void KVStringPair::setKey(const XMLCh* const newKey)
-{
-    setKey(newKey, XMLString::stringLen(newKey));
-}
-
-inline void KVStringPair::setValue(const XMLCh* const newValue)
-{
-    setValue(newValue, XMLString::stringLen(newValue));
-}
-
 inline void KVStringPair::setKey(  const XMLCh* const newKey
                                  , const XMLSize_t    newKeyLength)
 {
@@ -201,6 +191,16 @@ inline void KVStringPair::setValue(  const XMLCh* const newValue
     memcpy(fValue, newValue, (newValueLength+1) * sizeof(XMLCh)); // len+1 because of the 0 at the end
 }
 
+inline void KVStringPair::setKey(const XMLCh* const newKey)
+{
+    setKey(newKey, XMLString::stringLen(newKey));
+}
+
+inline void KVStringPair::setValue(const XMLCh* const newValue)
+{
+    setValue(newValue, XMLString::stringLen(newValue));
+}
+
 inline void KVStringPair::set(  const   XMLCh* const    newKey
                               , const   XMLCh* const    newValue)
 {
